### DISCUSSION TITLE
Suggestion: Compat with Another Furniture mod

### DISCUSSION THREAD

**Discussion by spunchbop121** [ThePandaOliver/Pandas-Extra-Details#25] [Ideas]
Shutter block:

![2023-11-16_00 31 35](https://github.com/PandaDap2006/Pandas-Extra-Details/assets/64424996/5feb8f41-0666-4c15-a79c-52db40e964f0)
![2023-11-16_00 32 00](https://github.com/PandaDap2006/Pandas-Extra-Details/assets/64424996/43a3a5b4-222e-440e-b7ec-7bbe21297a3f)
![2023-11-16_00 31 38](https://github.com/PandaDap2006/Pandas-Extra-Details/assets/64424996/9cd92932-5abc-414e-bc5a-397960754363)

**Comment by ThePandaOliver** [comment]
Thanks for the suggestion.
